src/subscription_manager/managercli.py

```diff
@@ -1854,6 +1854,8 @@ def __init__(self):
                                help=_("certificate file to import (can be specified more than once)"))
 
     def _validate_options(self):
+        if self.is_registered():
+            system_exit(os.EX_USAGE, _("Error: You may not import certs into a system that is registered to a subscription management service."))
```

Comment:
I would use "certificates" instead of "certs". To be consistent with the rest of the subman code base